tensorflow/java/src/gen/cc/ops/op_generator.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@asimshankar - in case it helps (this or the upcoming PR :-) [this is the draft of the change](https://github.com/tensorflow/tensorflow/compare/master...kbsriram:java-ops-3a-processor-bazel-draft) to kick off the annotation processor work.